lib/rmt/cli/base.rb

```diff
@@ -67,6 +67,12 @@ def handle_exceptions
         "The SCC credentials are not configured correctly in '/etc/rmt.conf'. You can obtain them from https://scc.suse.com/organization",
         RMT::CLI::Error::ERROR_SCC
       )
+    rescue RMT::Lockfile::ExecutionLockedError
+      pid = File.read(RMT::Lockfile::LOCKFILE_LOCATION)
+      raise RMT::CLI::Error.new(
+        "Process is locked by the application with pid #{pid}. Close this application or wait for it to finish before trying again\n",
```

Comment:
How about moving the entire logic of reading the lockfile pid into `RMT::Lockfile` and then creating the `RMT::Lockfile::ExecutionLockedError` with the message? Then `CLI::Base` doesn't have to worry about lockfile locations and things, it just outputs the message in contained in `RMT::Lockfile::ExecutionLockedError`.